src/main/java/com/spotify/folsom/reconnect/ReconnectingClient.java

```diff
@@ -62,13 +63,14 @@ public ReconnectingClient(final BackoffFunction backoffFunction,
                             final boolean binary,
                             final Executor executor,
                             final long timeoutMillis,
-                            final Charset charset) {
+                            final Charset charset,
+                            final Metrics metrics) {
```

Comment:
Yes, I was going to suggest that, but then I realized that we don't have a strong need to maintain such strong API guarantees anyway at this time, so perhaps it's better to keep it clean. (But we should probably bump the minor version instead of patch version)